scripts/plot_light_curve.py

```diff
@@ -23,16 +26,29 @@ def main(input_path, output):
     sed_type = lc.sed_type_plot_default
     y = getattr(lc, sed_type)
     y_errn, y_errp = lc._plot_get_flux_err(sed_type=sed_type)
+
     # This might be a bit overkill
     if np.any(~np.isnan(y)) or np.any(~np.isnan(y_errn)) or np.any(~np.isnan(y_errp)):
         lc.plot(ax=ax, sed_type=sed_type)
+        if blocks:
```

Comment:
`if blocks is not None:`

Reply:
Ok, if you prefer

Reply:
With the above change (or default?), this is the way